collect_app/src/androidTest/java/org/odk/collect/android/utilities/WebUtilsTest.java

```diff
@@ -25,31 +16,32 @@ public void setUp() throws Exception {
         server.enqueue(new MockResponse());
     }
 
-    @Test
-    public void httpRequests_shouldHaveUseragentHeader() throws Exception {
-        // when
-        doRequest("/some-path");
-
-        // then
-        RecordedRequest r = nextRequest();
-        assertEquals("GET /some-path HTTP/1.1", r.getRequestLine());
-        assertTrue(r.getHeader("User-Agent").matches("Dalvik/.* org.odk.collect.android/.*"));
-    }
+    //    @Test
```

Comment:
Thanks for the tip, I'll do that. there was a couple of tests in there where I wasn't sure what the value of them was. I'll take another look at them.

Reply:
I've had another look at the test. The reason I commented it out was because it uses httpclient directly to check that the User-agent is correct. I'm not sure on the exact value of this test, but also I will need to come to a http client agnostic solution moving forward.

Reply:
Sure, I guess some of these tests will be removed all together since they wont be necessary.